src/catalogue/catalogue.component.tsx

```diff
@@ -129,12 +144,18 @@ function Catalogue() {
     !catalogueId ? 'null' : catalogueId.replace('/', '')
   );
 
+  const catalogueCategoryNames: (string | undefined)[] =
+    catalogueCategoryData?.map((item) => item.name) || [];
```

Comment:
Not necessary for this PR but as an aside I think these can be combined so I created an issue for it https://github.com/ral-facilities/inventory-management-system/issues/227.